question29/reset-email.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link href="../css/style.css" rel="stylesheet" />
+        <link href="../css/common.css" rel="stylesheet" />
+        <link href="../css/form.css" rel="stylesheet" />
+        <script>
+            if (!localStorage.getItem("token")) window.location.href = "./login.html";
+          </script>
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <div id="js-wrapper" class="wrapper">
+            <h1 class="form_title">e-mail address Reset</h1>
+            <form method="post" id="form" novalidate>
+                <div>
+                    <ul>
+                        <li class="form_list">
+                            <label for="mail">新しいメールアドレス</label>
+                            <input type="email" id="mail" name="email" class="js-email" required>
+                            <span class="error" data-error="error"></span>
+                        </li>
+                        <li class="form_list">
+                            <label for="mail">新しいメールアドレス(確認用)</label>
+                            <input type="email" id="mail" name="email" class="js-email" required>
+                            <span class="error" data-error="error"></span>
+                        </li>
+                        <li class="form_list">
+                            <label for="password">パスワード</label>
+                            <button type="button" class="password_toggle_button" id="js-toggle-password-button" data-button="toggle-password-button">表示</button>
+                            <input type="password" autocomplete="new-password" id="new-password" name="password" class="js-password" data-input="password" required>
+                            <span class="error" data-error="error"></span>
```

Comment:
MUST
I thought it would be better to tie the label to the input. 
Also, since this is an existing password input field, `new-password` may not be appropriate.

Reply:
Thank you for your review! You are totally right! 
Labels and inputs were tied together. Also fixed autocomplete to current instead of new. Please confirm.

fixed commit : https://github.com/csakyo/homework/pull/48/commits/538a9cb7a6509631d1435367ef56779825e7ee23

Reply:
Q
Thanks for the fix!

Sorry for the addition. 
Is `class="js-password"` necessary?
Since `data-input` is given, I thought one or the other would be fine.

Reply:
Thank you for pointing it out! You are totally right! 
Removed classes that are no longer needed. Fixed so please confirm.

fiexe commit : https://github.com/csakyo/homework/pull/48/commits/b4cc181593f800c4470b45ce99a3f008cdefcb5b